action.yml

```diff
@@ -236,8 +237,8 @@ runs:
             log_info "Using bootstrap cosign to verify keyless signature of desired cosign version"
             "./${cosign_executable_name}" verify-blob --certificate-identity=keyless@projectsigstore.iam.gserviceaccount.com --certificate-oidc-issuer=https://accounts.google.com --bundle "${keyless_signature_file}" "cosign_${input_cosign_release}"
 
-            if is_version_ge "3.0.2" "$version_num"; then
-              # we're trying to get something greater than or equal to v3.0.2
+            if is_version_ge "3.0.3" "$version_num"; then
```

Comment:
We can keep this as 3.0.2 - this is the new format for signatures going forward.

Reply:
dne